ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -44,10 +46,29 @@ def collect_must_gather(self):
         """
         Collect ocs_must_gather and copy the logs to a temporary folder.
 
+        Verify the output of mg command does not contain "cannot stat 'jq'"
+        https://bugzilla.redhat.com/show_bug.cgi?id=1974959
```

Comment:
I think we dont need to add these details here in the function doc string, maybe we can add these as log.info and comments inside the function

Reply:
@PrasadDesala with bz link?